knative-kubecon/README.md

```diff
@@ -7,39 +7,36 @@ offer:
 Build mechanic underneath).
 2. The built image is then deployed as a Knative Service, which means it scales automatically, even down
 to nothing as we'll see.
-3. We wire an EventSource emitting IoT events from the Newcastle University to our application through 
+3. We wire an EventSource emitting IoT events from the heartbeat source to our application through 
 Knative's Eventing capabilities.
-4. We'll roll out a new version via a canary release.
+4. We'll roll out a new version by using a canary release.
 5. The application will be scaled according to the needs of the incoming traffic.
 
 ## 0. Setting up an OpenShift cluster
 
-Before we get started, let's setup an environment. This setup guide assumes the usage of `minishift` and the
-setup scripts are tailored to facilitate that. To setup a fresh minishift cluster with OLM, Istio and all parts
-we need from Knative installed, run the following scripts:
+Before we get started, let's setup an environment. This setup guide, assumes the usage of OpenShift 4.0 (OCP 4) and the setup scripts are tailored to facilitate that. To setup a cluster with Istio and the installed Knative components, run the following scripts:
 
-```bash
-git clone git@github.com:openshift-cloud-functions/knative-operators.git
-./knative-operators/etc/scripts/install.sh
-./scripts/preload.sh
-```
+  ```bash
+  git clone https://github.com/openshift-cloud-functions/demos.git
+  cd demos/knative-kubecon
```

Comment:
I believe we need to clone and cd into the demos repo after we run the cluster setup script.